MonoGame.Framework/Input/Mouse.Windows.cs

```diff
@@ -31,7 +36,7 @@ private static void PlatformSetPosition(int x, int y)
             PrimaryWindow.MouseState.X = x;
             PrimaryWindow.MouseState.Y = y;
             
-            var pt = Window.PointToScreen(new System.Drawing.Point(x, y));
+            var pt = _window.PointToScreen(new System.Drawing.Point(x, y));
```

Comment:
> I was thinking more along lines allow Mouse.SetPosition to also take GameWindow window as an argument.

I don't think we need to do this yet...  let's wait for someone to need it first.

Like I think I am the only person to ever use the multiple `GameWindow` feature.  I specifically added it for this project:

https://www.youtube.com/watch?v=yz6kkqHGEhM   #BuiltWithMonoGame

Specifically because we have games with 3 monitors and touch screens on two of them.